src/events/custom/chatPoint.ts

```diff
@@ -107,7 +107,7 @@ export default class ChatPointEvent {
         }
 
         // If contains URLs
-        if (message.cleanContent.match(/https?:\/\/[^.]\.[^/]+/g)) {
+        if (message.cleanContent.match(/https?:\/\/[^.]*.[^/]*/g)) {
```

Comment:
Ye I forgor `+` quantifier.
But it should be `/https?:\/\/[^.]+\.[^/]+/g`
The valid url should contains at least 2 character chains splitted by dot chararacter `\.` (not `.`)

```suggestion
        if (message.cleanContent.match(/https?:\/\/[^.]+\.[^/]+/g)) {
```